package.json

```diff
@@ -77,6 +77,6 @@
   },
   "dependencies": {
     "fp-ts": "^2.14.0",
-    "zod": "^3.21.4"
+    "zod": "^3.23.8"
```

Comment:
The fix here is correct but `zod` shouldn't have been listed as a dependency in the first place. It should have been a peer-dependency instead specifying any version above 3.

Reply:
I may as well make that change now